src/huggingface_hub/utils/__init__.py

```diff
@@ -13,3 +13,47 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License
+
+import subprocess
+from typing import List
+
+from .logging import get_logger
+
+
+logger = get_logger(__name__)
+
+
+def run_subprocess(
```

Comment:
I've had bad experiences in including functions in `__init__` files since soon it'll cause some circular import issues. Could we move this to a `utils/_subprocess.py` file (I don't really feel strongly about the file name)?

Reply:
Yes, sounds good. Will move it around!